src/metrics-schema.ts

```diff
@@ -1,27 +0,0 @@
-import joi from 'joi';
-
-export const countSchema = joi
-    .object()
-    .options({ stripUnknown: true })
-    .keys({
-        yes: joi.number().min(0).empty('').default(0),
-        no: joi.number().min(0).empty('').default(0),
-        variants: joi.object().pattern(joi.string(), joi.number().min(0)),
```

Comment:
@olav If I understand correctly, then this should map to something like this in OpenAPI, right:

```js
  variants: {
      type: 'object',
      additionalProperties: {
          type: 'integer',
          minimum: 0,
      },
  }
```

Reply:
The `default` might be tricky. I don't think OpenAPI will modify the request to fill inn defaults. Could do it manually. :thinking:

Reply:
Oh, right, I hadn't even thought of the default! I was thinking about the `variants` prop, but that's a good point!